# Copyright (c) Facebook, Inc. and its affiliates.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# This hangs on CI and is not needed #9116
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,7 +84,6 @@ endif()
 include(GNUInstallDirs)
 include(FetchContent)
 include(cmake/code_coverage.cmake)
-include(cmake/sanitizer.cmake)
 include(cmake/clear_variable.cmake)

 # So CMake can find FindMbedTLS.cmake
